sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -2138,6 +2157,7 @@ pi_result piQueueCreate(pi_context Context, pi_device Device,
   try {
     *Queue =
         new _pi_queue(ZeCommandQueue, Context, Device, ZeCommandListBatchSize);
+    (*Queue)->PiQueueProperties = Properties;
```

Comment:
add this to constructor instead

Reply:
We call new _pi_queue in piextQueueCreateWithNativeHandle as well. But there is no Properties coming into that function. What do we do there? Thanks.

Reply:
We will have to choose something as a default, I guess OOO since that is the default SYCL's choice. If needed we could add an extension for users' to specify properties with the interop API later on.

Reply:
Please do have it in the constructor.

Reply:
sorry, missed this comment, done.